scripts/community_scoring.py

```diff
@@ -42,14 +49,14 @@ def potential_fraud_subgraph(unfrozen_graph):
     return unfrozen_graph
 
 
-def potential_fraud_score(graph, weight=0.7):
+def potential_fraud_score(graph, weight=1):
     ''' 4. calculate the total number of potential fraud count_fraud_indicators
         5. calculate total number of possible relationships
         6. return the community fraud score
         '''
     sum_fraud_edges = 0
     for node1, node2, data in list(graph.edges(data=True)):
-        if data['properties']['weight'] == 2:
+        if data['properties']['weight'] <= 2:
```

Comment:
This should be >= because we are trying to count any potential fraud edges where the weight is 2 or more.